lib/readline.js

```diff
@@ -1133,5 +1130,6 @@ module.exports = {
   createInterface,
   cursorTo,
   emitKeypressEvents,
-  moveCursor
+  moveCursor,
+  ESCAPE_CODE_TIMEOUT : 500 // GNU readline library - keyseq-timeout is 500ms (default)
```

Comment:
If we're going to do this, passing `escapeCodeTimeout` as an option to `readline.createInterface()` would seem to be the better approach.